src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.model.dto.request;
+
+import lombok.Data;
+
+@Data
+public class UserRequestDto {
+    private String email;
+    private String password;
```

Comment:
Use @ToString.Exclude annotation. We should not provide password info in toString method